src/mobot_client/air_drop_session.py

```diff
@@ -130,54 +121,49 @@ def handle_airdrop_session_ready_to_receive(self, message, drop_session):
                     drop_session.drop.conversion_rate_mob_to_currency
                 )
                 self.logger.info(f"Sending customer {drop_session.customer} initial coin amount...")
-                self.payments.send_mob_to_customer(drop_session.customer, message.source, amount_in_mob, True)
+                self.payments.send_mob_to_customer(drop_session.customer, amount_in_mob, True)
                 self.messenger.log_and_send_message(
                     drop_session.customer,
-                    message.source,
                     ChatStrings.AIRDROP_INITIALIZE.format(
                         amount=amount_in_mob.normalize(),
                         symbol=drop_session.drop.currency_symbol,
                         value=value_in_currency
                     )
                 )
+                self.logger.info(f"Sending customer {drop_session.customer} initial coin amount...")
+                self.payments.send_mob_to_customer(drop_session.customer, message.source, amount_in_mob, True)
                 self.messenger.log_and_send_message(
                     drop_session.customer,
-                    message.source,
                     ChatStrings.PAY_HELP
                 )
                 drop_session.state = SessionState.WAITING_FOR_PAYMENT
         else:
             self.messenger.log_and_send_message(
                 drop_session.customer,
-                message.source,
                 ChatStrings.YES_NO_HELP
             )
         drop_session.save()
 
     def handle_drop_session_waiting_for_bonus_transaction(self, message, drop_session):
-        self.logger.info("----------------WAITING FOR BONUS TRANSACTION------------------")
+        print("----------------WAITING FOR BONUS TRANSACTION------------------")
```

Comment:
I would've thought we'd want log output or no output, what's the thinking on using `print` here?